tests/e2e/tests/kiali/kiali_test.go

```diff
@@ -0,0 +1,70 @@
+package simple
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"testing"
+
+	"istio.io/istio/pkg/log"
+	"istio.io/istio/tests/e2e/framework"
+	"istio.io/istio/tests/util"
+)
+
+type testConfig struct {
+	*framework.CommonConfig
+}
+
+var (
+	tc        *testConfig
+	testFlags = &framework.TestFlags{
+		Ingress: true,
+		Egress:  true,
+	}
+)
+
+func init() {
+	testFlags.Init()
+	flag.Parse()
+}
+
+func TestMain(m *testing.M) {
+	if err := framework.InitLogging(); err != nil {
+		panic("cannot setup logging")
+	}
+	if err := setTestConfig(); err != nil {
+		log.Error("could not create TestConfig")
+		os.Exit(-1)
+	}
+	os.Exit(tc.RunTest(m))
+}
+
+func TestKialiPod(t *testing.T) {
+	ns := tc.Kube.Namespace
+	// Get the 2 pods
+	podList, err := getPodList(ns, "app=kiali")
+	if err != nil {
+		t.Fatalf("kubectl failure to get pods %v", err)
+	}
+	if len(podList) != 1 {
+		t.Fatalf("Unexpected to get %d pods when expecting 1. got %v", len(podList), podList)
+	}
+}
+
+func getPodList(namespace string, selector string) ([]string, error) {
```

Comment:
can you add this function somewhere in the the infrastructure codebase?

Reply:
@sdake is dashboard_test.go a good place to Kiali Tests?

Regards,
Guilherme Baufaker